### PR TITLE
Fix "Create Form Builder Form attribute"

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | master, 4.6, 4.5
| Edition       | Experience, Commerce

- Use dedicated `assets/js/` folder instead of generic `public/js/` folder
- Fix configuration key from `ibexa_form_builder_fields` to `ibexa_form_builder.fields`
- Fix attribute name missing space
- Frame illustration
- Add more illustrations of the resulting attribute

Comparison:
- [Old/current](https://doc.ibexa.co/en/latest/content_management/forms/create_form_attribute/)
- [New](https://ez-systems-developer-documentation--2402.com.readthedocs.build/en/2402/content_management/forms/create_form_attribute/)

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/content_management/forms/img/checkbox_with_richtext_description-preview.png
Comment (julitafalcondusza) [inline docs/content_management/forms/img/checkbox_with_richtext_description-preview.png:1]: I'd rather see full Form window, more context is needed here (plus it doesn't look good cut like that).

File: docs/content_management/forms/create_form_attribute.md [hunk 1]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
...
 You should be able to see the new section in the list of available fields:
 
-![New form](new_form.png)
+![New form attribute](checkbox_with_richtext_description-item.png)
+
+When the attribute settings are edited, the "Description" field is a richtext textarea.
+
+![Attribute settings](checkbox_with_richtext_description-edit.png)
+
+The richtext toolbar appears when "Description" field is focused.
```
Comment (adriendupuis) [inline docs/content_management/forms/create_form_attribute.md:127]:
```suggestion
The richtext toolbar appears when "Description" attribute is focused.
```

File: docs/content_management/forms/create_form_attribute.md [hunk 2]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
 
 You should be able to see the new section in the list of available fields:
 
-![New form](new_form.png)
+![New form attribute](checkbox_with_richtext_description-item.png)
+
+When the attribute settings are edited, the "Description" field is a richtext textarea.
```
Comment (adriendupuis) [inline docs/content_management/forms/create_form_attribute.md:123]:
```suggestion
The field settings have a richtext "Description" attribute.
```
or
```suggestion
The field settings' "Description" attribute is richtext.
```

File: docs/content_management/forms/create_form_attribute.md [hunk 3]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
 
 You should be able to see the new section in the list of available fields:
 
-![New form](new_form.png)
+![New form attribute](checkbox_with_richtext_description-item.png)
```
Comment (adriendupuis) [inline docs/content_management/forms/create_form_attribute.md:121]:
```suggestion
![New form field](checkbox_with_richtext_description-item.png)
```

File: docs/content_management/forms/create_form_attribute.md [hunk 4]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
...
+
+When the attribute settings are edited, the "Description" field is a richtext textarea.
+
+![Attribute settings](checkbox_with_richtext_description-edit.png)
```
Comment (adriendupuis) [inline docs/content_management/forms/create_form_attribute.md:125]:
```suggestion
![Field settings](checkbox_with_richtext_description-edit.png)
```

File: docs/content_management/forms/create_form_attribute.md [hunk 5]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
...
+
+The richtext toolbar appears when "Description" field is focused.
+
+![Richtext toolbar](checkbox_with_richtext_description-focus.png)
+
+The preview show the checkbox, its label, and its formatted richtext.
+
+![Attribute preview](checkbox_with_richtext_description-preview.png)
```
Comment (adriendupuis) [inline docs/content_management/forms/create_form_attribute.md:133]:
```suggestion
![Field preview](checkbox_with_richtext_description-preview.png)
```

File: docs/content_management/forms/create_form_attribute.md (hunk 2 above)
Comment (julitafalcondusza) [inline docs/content_management/forms/create_form_attribute.md:123]: When editing settings, the "Description" attribute has the Rich Text input.

**Review by julitafalcondusza — CHANGES_REQUESTED** [review]
Please see my comments.

File: docs/content_management/forms/create_form_attribute.md (hunk 2 above)
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md:123]:
```suggestion
When editing settings, the "Description" attribute has the Rich Text input.
```

File: docs/content_management/forms/create_form_attribute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md:127]: It doesn't appear on hover. You need to click into the field to get the text cursor and the richtext toolbar.

[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md:127]: @julitafalcondusza What about the following?:
```suggestion
When you enter the "Description" field, the Rich Text toolbar appears.
```

File: docs/content_management/forms/create_form_attribute.md [hunk 6]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
...
-![New form](new_form.png)
+![New form attribute](checkbox_with_richtext_description-item.png)
+
+When the attribute settings are edited, the "Description" field is a richtext textarea.
+
+![Attribute settings](checkbox_with_richtext_description-edit.png)
+
+The richtext toolbar appears when "Description" field is focused.
+
+![Richtext toolbar](checkbox_with_richtext_description-focus.png)
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md:129]: If I change Richtext to Rich Text, I should do it every where in the page.

File: docs/content_management/forms/create_form_attribute.md [hunk 7]
```diff
@@ -118,4 +118,16 @@ In the main menu, click **Content** -> **Forms** -> **Create content**, and sele
...
+
+The preview show the checkbox, its label, and its formatted richtext.
```
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md:131]:
```suggestion
The preview displays the formatted text along with the checkbox and its label.
```

File: docs/content_management/forms/img/checkbox_with_richtext_description-preview.png
Comment (adriendupuis) [reply docs/content_management/forms/img/checkbox_with_richtext_description-preview.png:1]: I updated with a large view in 840628fc487004ddba0231abc556c82fe0d0ad6f

File: docs/content_management/forms/create_form_attribute.md (hunk 2 above)
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md]: Done in f8fea01e3584f34c2d0b467d3b343af929c308f5

File: docs/content_management/forms/create_form_attribute.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (adriendupuis) [reply docs/content_management/forms/create_form_attribute.md:129]: We may have a consistency issue

File: code_samples/forms/custom_form_attribute/config/packages/form_attribute_config.yaml [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ibexa_form_builder:
     fields:
         checkbox_with_richtext_description:
-            name: Checkbox with RichtextDescription
+            name: Checkbox with Richtext Description
```
Comment (adriendupuis) [inline code_samples/forms/custom_form_attribute/config/packages/form_attribute_config.yaml:4]:
```suggestion
            name: Checkbox with Rich Text Description
```

**Review by julitafalcondusza — APPROVED** [review]

**Pull request merged**